src/_index.xs

```diff
@@ -0,0 +1,69 @@
+const knex = require('knex')(process.env.DATABASE_URL);
+const knexLog = require('knex-log')(knex, { tableName: 'image-search-logs', columnName: 'data' });
+
+// TODO: update knex-log to allow the setting of highWaterMark and batchSize
+// -- highWaterMark is the watermark of the underlying readable stream
+// -- batchSize is the number of items pg-query-stream pushes on at a time (defaults to 100)
+// This means that if you have a highWaterMark value less than 100 you probably want to
+// adjust the batchSize as well... (ie. with highWaterMark of 16 and batchSize of 100, the
+// readable stream will hold between 100 and 115 items assuming there are still more entries
+// to be returned from the knex query.
+//
+// Note: a highWaterMark of 16 with a batchSize of 100 is going to be the default behaviour
+// @see https://nodejs.org/api/stream.html#stream_new_stream_readable_options for readable
+// stream options (specifically the default values).
+
+// const amqp = require('amqplib');
+let conn;
+
+// TODO: cs-media-search-logger-staging and cs-image-crawl-staging use differend DBs...
+// they should probably both use the cs-media-search-logger-staging one like we do in live.
```

Comment:
My intention has been that there was no real reason to have them in the same DB - we should be able to split the live stuff if we needed to.